redisinsight/ui/src/components/navigation-menu/NavigationMenu.tsx

```diff
@@ -170,11 +156,8 @@ const NavigationMenu = () => {
       dataTestId: 'analytics-page-btn',
       connectedInstanceId,
       isActivePage: isAnalyticsPath(activePage),
-      getClassName() {
-        return cx(navigationButtonStyle, { [styles.active]: this.isActivePage })
-      },
       getIconType() {
-        return this.isActivePage ? SlowLogActiveIcon : SlowLogIcon
+        return this.isActivePage ? SlowLogIcon : SlowLogActiveIcon
```

Comment:
Is there a reason behind the inconsistency of the names of the icons?

| Icon | Default | Active |
| -------- | -------- | ------- |
| Browser | dark | light |
| Workbench | dark | light |
| SlowLog | light | dark |
| PubSub | light | dark |
| Pipeline Statistics | light | dark |

I have checked it in the UI and it looks fine, just the names look out of sync and I wonder whether it was intentional at some point?